technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -28,8 +28,8 @@
       <term name="issue" form="short">H.</term>
       <term name="translator" form="verb-short">deutsch von</term>
       <term name="presented at">gehalten auf</term>
-      <term name="no date" form="short">o.&#8239;J.</term>
-      <term name="et-al">u.&#8239;a.</term>
+      <term name="no date" form="short">o. J.</term>
+      <term name="et-al">u. a.</term>
```

Comment:
> I use the visual csl editor a lot, it automatically converts decimal code to special characters and I always thought this was a good thing.

Ah, I didn't know it did that. It might be better if it we change it to not do that. Do you happen to know when the visual editor does that conversion? As soon as you type in the codes, or only when you download the edited style?

> For now, could you merge anyway and have your script do the dirty work?

Will do.